eng/Versions.props

```diff
@@ -299,6 +299,7 @@
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
       https://dotnet.myget.org/F/dotnet-corefxtestdata/api/v3/index.json;
       https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
+      https://dotnet.myget.org/F/experimental-vs-packages/api/v3/index.json;
```

Comment:
Why the new feed?

Reply:
@jasonmalinowski this is the best we found we @heejaechang . Please let me know if we can use an existing one. Which one?

Reply:
Why aren't these in the editor feed? If it can't be there then we can put it in roslyn-tools.

Reply:
It is not directly made by Editor. It is Microsoft.VisualStudio.Utilities. See the email thread.
All Editor binaries are properly published in existing Editor feeds.

Reply:
Ah, then this should be in roslyn-tools.

Reply:
For those following along at home, I uploaded the package.